### PR TITLE
Feat/notify lifebank nearby

### LINKED ISSUE #698: Emails flow: lifebanks users
Dep: #684

Front-end: @steph222 
Backend: @leisterfrancisco

### LINKED ISSUE #699: Emails flow: sponsors users
Dep: #684

Front-end: @steph222 
Backend: @leisterfrancisco

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #698
Resolves #699 

### What does this PR do?
Notify all sponsor and donor user when a new lifebank is registered near of them

### How should this be tested?
- Make run
- Register new lifebank
- Check email for user near of that lifebank coordinates

### CODE REVIEW

File: hapi/src/api/account.api.js [hunk 1]
```diff
@@ -150,13 +150,100 @@ const createLifebank = async ({
...
+
+  if (donorsWithLocation && donorsWithLocation.length > 0) {
+    donorsWithLocation.forEach((donor) => {
+      if (isCoordinateInsideBox(lifebank.geolocation, donor.location))
+        mailApi.sendNewLifebankRegistered(
+          donor.email,
+          'New lifebank near you!',
+          'There’s a new lifebank near you! Check its profile <a href="https://lifebank/info/' +
+            lifebankAccount +
+            '">here</a>. Visit them soon and help save lives!<br><br>You can unsubscribe anytime from receiving these communications <a href="http://lifebank/cancel-email-subscription/' +
```
Comment (leisterfrancisco) [inline hapi/src/api/account.api.js:180]: Pending to hapi-i18n

File: hapi/src/api/account.api.js [hunk 2]
```diff
@@ -150,13 +150,100 @@ const createLifebank = async ({
...
+    role: { _eq: 'sponsor' }
+  })
+
+  if (donorsWithLocation && donorsWithLocation.length > 0) {
+    donorsWithLocation.forEach((donor) => {
+      if (isCoordinateInsideBox(lifebank.geolocation, donor.location))
+        mailApi.sendNewLifebankRegistered(
+          donor.email,
+          'New lifebank near you!',
+          'There’s a new lifebank near you! Check its profile <a href="https://lifebank/info/' +
```
Comment (leisterfrancisco) [inline hapi/src/api/account.api.js:178]: hapi-i18n

File: hapi/src/api/account.api.js [hunk 3]
```diff
@@ -150,13 +150,100 @@ const createLifebank = async ({
...
+            lifebankAccount +
+            '">here</a>. Visit them soon and help save lives!<br><br>You can unsubscribe anytime from receiving these communications <a href="http://lifebank/cancel-email-subscription/' +
+            lifebankAccount +
+            '">here</a>.'
```
Comment (leisterfrancisco) [inline hapi/src/api/account.api.js:182]: hapi-i18n

File: hapi/src/api/account.api.js [hunk 4]
```diff
@@ -150,13 +150,100 @@ const createLifebank = async ({
...
+        sponsorProfile.location &&
+        isCoordinateInsideBox(
+          lifebank.geolocation,
+          JSON.parse(sponsorProfile.location)
+        )
+      )
+        mailApi.sendNewLifebankRegistered(
+          sponsorProfile.email,
+          'New lifebank near you!',
+          'There’s a new lifebank near you! Check its profile <a href="http://lifebank/info/' +
```
Comment (leisterfrancisco) [inline hapi/src/api/account.api.js:201]: hapi-i18n

File: hapi/src/api/account.api.js [hunk 5]
```diff
@@ -150,13 +150,100 @@ const createLifebank = async ({
...
+            lifebankAccount +
+            '">here</a>. Visit them soon and help save lives!<br><br>You can unsubscribe anytime from receiving these communications <a href="http://lifebank/cancel-email-subscription/' +
+            lifebankAccount +
+            '">here</a>.'
```
Comment (leisterfrancisco) [inline hapi/src/api/account.api.js:205]: hapi-i18n

**Review by Edgar097 — APPROVED** [review]
image
LGTM good job!

**Comment by leisterfrancisco** [comment]
> image
> LGTM good job!

@Edgar097 thanks for your feedback, we can open another issue to solve this image rendering

**Review by steph222 — APPROVED** [review]

**Pull request merged**